doc: loadBalancerIP usage (#9)
scaleway/scaleway-cloud-controller-manager: docs/loadbalancer-examples.md

```bash
curl -i http://51.15.224.149
```

## Deploy a loadbalancer with specific IP

> Instead of creating a loadbalancer service with an ephemeral IP (which change each time your delete/create the service), you can use a reserved loadbalancer address.

Reviewer: ```suggestion
Instead of creating a loadbalancer service with an ephemeral IP (which changes each time your delete/create the service), you can use a reserved loadbalancer address.
```

> _Warning: LoadBalancer ips are different from Instance ips and are not echangeable._

Reviewer: ```suggestion
_Warning: LoadBalancer ips are different from Instance ips. You cannot use an instance ip on a load-balancer and vice versa._
```

To reserve a loadbalancer IP through API :
```bash
curl -X POST "https://api.scaleway.com/lb/v1/regions/$REGION/ips" -H "X-Auth-Token: $SECRET_KEY" -H "Content-Type: application/json" \

> -d "{\"organization_id\":\"$ORGANIZATION_ID\"} | jq -r .ip_address"

Reviewer: Stay consistent with the variables defined here: https://www.terraform.io/docs/providers/scaleway/index.html#arguments-reference

```

Now specify this IP address to spec.loadBalancerIP field of the service

```yaml
kind: Service
apiVersion: v1
metadata:
  name: demo-nginx
spec:
  selector:
    app: demo-nginx
  type: LoadBalancer
  ports:
  - name: http
    port: 80
    targetPort: 80
  loadBalancerIP: SCW.LB.IP.ADDR
```

> ## Convert a ephemeral IP into reserved IP

Reviewer: ```suggestion
## Convert an ephemeral IP into reserved IP
```

> It's possible to keep an ephemeral address (converting into reserved address) by patching the associated service.

Reviewer: ```suggestion
It's possible to keep an ephemeral address (converting into a reserved address) by patching the associated service.
```
